### PR TITLE
fix: very small token values are being displayed in scientific format

### LINKED ISSUE #19317: Small numbers are displayed in scientific notation in community permissions
Example of community https://status.app/c/CxWACikKDlNlcG9saWEgY29tbSA2EgdlZHNhYXNkGAEiByM0MzYwREYqAwULKgM=#zQ3shsAnHv4WJUa2d5QWAQa5NKtQoes8sCjnN8VH3dQ78j2L4

This community has a permission: user should hold 0.000001 ETH to become a member

**Preconditions:** mobile user is switched to testnet Sepolia

**Steps:** 

1. Open given community on mobile
2. Pay attention at how permission number is displayed

**Expected result:** 0.000001 ETH is displayed

**Actual result:** permission is displayed in scientific notation (1e-06 ETH)

photo_2024-03-18 18 23 44

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by ajayesivan**
fixes #19317 

status: ready

### CODE REVIEW

**Review by jo-mut — APPROVED** [review]

File: src/status_im/subs/communities.cljs [hunk 1]
```diff
@@ -306,12 +306,13 @@
   [checking-permissions?
    token-images
    {:keys [satisfied criteria]}]
-  (let [sym    (:symbol criteria)
-        amount (:amount criteria)]
+  (let [sym         (:symbol criteria)
+        amountInWei (:amountInWei criteria)
```
Comment (ibrkhalil) [inline src/status_im/subs/communities.cljs]: Kebab case?

**Review by ibrkhalil — APPROVED** [review]

**Comment by pavloburykh** [comment]
@ajayesivan thank you for the fix! Ready for merge. Please, cherrypick this fix into release branch as well. cc @cammellos

**Comment by ajayesivan** [comment]
@pavloburykh Thank you for testing the PR! I will merge and cherrypick now 👍

**Comment by ajayesivan** [comment]
> @ajayesivan thank you for the fix! Ready for merge. Please, cherrypick this fix into release branch as well. cc @cammellos

Done

**Pull request merged**